### PR TITLE
Fix set and get view model method code

### PR DESCRIPTION
**Pull request by pawan**
## Purpose of this pull request

This pull request (PR) is fix for issue #8493

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/extension-dev-guide/view-models.html

### CODE REVIEW

**Comment by pawan** [comment]
Adobe CLA sign

**Comment by hostep** [comment]
@pawan, thanks, but this duplicates https://github.com/magento/devdocs/pull/8459 I believe, maybe you can collaborate on that ticket to get everything explained better?

File: src/guides/v2.3/extension-dev-guide/view-models.md [hunk 1]
```diff
@@ -139,3 +139,5 @@ $postArray = $viewModel->getPostData(
     ['product' => $_item->getEntityId()]
 );
 ```
+{:.bs-callout-info}
+If you use `argument name="view_model"` you can access it using `$viewModel = $block->getViewModel();` Or You can use `argument name="viewModel"` and can access `$viewModel = $block->getData('viewModel');`
```
Comment (BarnyShergold) [inline src/guides/v2.3/extension-dev-guide/view-models.md:143]: IF this is true then the to changes above are mutually exclusive - only one of them should have been changed.

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/extension-dev-guide/view-models.md (hunk 1 above)
Comment (pawan) [reply src/guides/v2.3/extension-dev-guide/view-models.md:143]: yes, It depends on implementation! either we can use Getter/Setter OR we can get Data of argument!

Comment (pawan) [reply src/guides/v2.3/extension-dev-guide/view-models.md:143]: @BarnyShergold  First change I made as per example and second change is from actual changes from file [https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Catalog/view/frontend/layout/catalog_product_view.xml#L178](url)

**Comment by dobooth** [comment]
Thanks for this, @pawan. Since we have 3 PRs around this fix, feels proper to take the first one. Thanks for contributing, though. Nice to see so many eyes on the docs. We will take #8459 for the fix.

**Comment by pawan** [comment]
OK, why it is showing 2 closed pull request at [https://opensource.magento.com/profile/pawan/contribution_statistic](url)

**Pull request closed without merging**